src/comments/CommentItem.scss

```diff
@@ -1,22 +1,55 @@
 @import '../styles/colors';
 
 .CommentItem {
-  word-break: break-all;
-  padding: 1rem 1.5rem;
+  word-break: keep-all;
+
   a.active {
     i {
       color: $primary-color;
     }
   }
 }
 
+.CommentItem__content {
+  margin: 0.5em 0;
+  display: flex;
+
+  &.CommentItem__content--level-1 {
```

Comment:
No need to nest modifiers inside elements, define them below the elements separately. The whole point of writing the full name in modifiers is to keep the CSS structure flat. This way the final CSS file would also be shorter since selectors are not long and nested.